### PR TITLE
5 min beginners tutorial

### PR DESCRIPTION
**Pull request by nelsonic**
adding a super basic example to lower the learning curve for _complete_ beginners.
@andrewMacmurray please take a look at the update, leave any comments if think any changes are required and _merge_ if you're happy. thanks! 👍 

[image](https://twitter.com/iamdevloper/status/753716544949981184 "CSS is easy? - click to see original tweet")

### CODE REVIEW

**Comment by nelsonic** [comment]
image

**Comment by nelsonic** [comment]
rob-ybu-know-css-come-quick-to-disarm-the-bomb-2466256

File: examples/buttons.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Buttons - Tachyons Tutorial</title>
+    <!-- Pure CSS is a minimalist CSS file we have included to make things look nicer -->
```
Comment (andrewMacmurray) [inline examples/buttons.html]: Is this comment supposed to be here? Or is the `Pure CSS` file missing? (same with the other 3 html files)

**Review by andrewMacmurray — COMMENTED** [review]
This looks great, you've put a lot of work into it! Just the one comment for clarifying the pure css thing but other than that it's good to go 🎉

**Comment by nelsonic** [comment]
@andrewMacmurray well-spotted! thanks! 
Removed the instances of `Pure CSS` comment from files. 
Please merge if ok. thanks! 🎉

**Pull request merged**